poc/vdaf.py

```diff
@@ -221,9 +220,21 @@ def run_vdaf(Vdaf,
             outbound = []
             for j in range(Vdaf.SHARES):
                 out = Vdaf.prep_next(prep_states[j], prep_msg)
-                if i < Vdaf.ROUNDS - 1:
-                    (prep_states[j], out) = out
+                (prep_states[j], out) = out
                 outbound.append(out)
+            # REMOVE ME
+            for prep_share in outbound:
+                prep_test_vec['prep_shares'][i+1].append(prep_share.hex())
+
+        # The final outputs of prepare phase are the output shares.
+        prep_msg = Vdaf.prep_shares_to_prep(agg_param,
+                                            outbound)
+        # REMOVE ME
+        prep_test_vec['prep_messages'].append(prep_msg.hex())
+        outbound = []
```

Comment:
This should have another blank line, only one statement is excluded from the document.
```suggestion
        # REMOVE ME
        prep_test_vec['prep_messages'].append(prep_msg.hex())

        outbound = []
```